types/TextArea/TextArea.svelte.d.ts

```diff
@@ -8,7 +8,7 @@ export interface TextAreaProps extends RestProps {
    * Specify the textarea value
    * @default ""
    */
-  value?: string;
+  value?: null | string;
```

Comment:
These files are generated and should not to be edited manually.

The property would need a `@type {...}` annotation in the source code.

For reference, the property in `TextInput`:
https://github.com/carbon-design-system/carbon-components-svelte/blob/21b1d78d8ebda608d78c1ab22833c36138f9ef3c/src/TextInput/TextInput.svelte#L13-L20

But, I do not think changing this at all makes sense.

The input only has `null` to accommodate `type="number"` (see documentation above).

Reply:
Ok I first wanted to add it there, but then I was confused how it get the type if there is nothing defined before.

At least for the component it does not make a different if it is null or undefined, doesn't it? To put values in the user mabybe have to change the type or do a cast, without the need to, because the textarea component can handle null values as it do for undefined. That is currently our problem.